src/lamplib/tasks/__init__.py

```diff
@@ -33,6 +33,7 @@ def cmake(context, toolchain_dir, env, cmdline_cmake_args):
     cmake_cmd += [
         "-DCMAKE_PREFIX_PATH={}".format(cmake_prefix_path),
         "-DCMAKE_TOOLCHAIN_FILE={}".format(cmake_toolchain_file),
+        "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON",
```

Comment:
Does this cause compile_commands.json to be emitted? If so, can you add it to `.gitignore`?

Reply:
It gets dumped in `build` so is ignored anyway! I actually didn't mean to include that I just use it locally for pylinter; I can remove it if anyone prefers it not be there.

Reply:
Hey this is a really nice change I wish we'd thought of it earlier. Huge 👍🏽